src/frontend/src/pages/WorkPackageDetailPage/WorkPackageViewContainer/WorkPackageDetails.tsx

```diff
@@ -3,64 +3,107 @@
         </Grid>
 
-        <Grid item xs={4} md={4}>
-          <DetailDisplay label="Progress" content={percentPipe(workPackage.progress)} paddingRight={2} />
+        <Grid item display="flex" alignItems="center" xs={12} sm={6} md={3}>
+          <ScheduleIcon sx={{ mr: 2 }} />
+          <DetailDisplay label="Start Date" content={datePipe(workPackage.startDate)} paddingRight={1} />
         </Grid>
 
-        <Grid item xs={4} md={4}>
-          <DetailDisplay label="Manager" content={fullNamePipe(workPackage.projectManager)} paddingRight={2} />
+        <Grid item display="flex" alignItems="center" xs={12} sm={6} md={3}>
+          <StackedLineChartIcon sx={{ mr: 2 }} />
+          <DetailDisplay label="Progress" content={percentPipe(workPackage.progress)} paddingRight={1} />
         </Grid>
 
-        <Grid item xs={4} md={4}>
-          <DetailDisplay label="End Date" content={datePipe(workPackage.endDate)} paddingRight={2} />
+        <Grid item display="flex" alignItems="center" xs={12} sm={6} md={3}>
+          <ScheduleIcon sx={{ mr: 2 }} />
+          <DetailDisplay label="Duration" content={weeksPipe(workPackage.duration)} paddingRight={1} />
         </Grid>
 
-        <Grid item xs={4} md={4}>
-          <DetailDisplay label="Expected Progress" content={percentPipe(workPackage.expectedProgress)} paddingRight={2} />
+        <Grid item display="flex" alignItems="center" xs={12} sm={6} md={3}>
+          <Work sx={{ mr: 2 }} />
+          <DetailDisplay label="Manager" content={fullNamePipe(workPackage.projectManager)} paddingRight={1} />
         </Grid>
 
-        <Grid item xs={4} md={4}>
-          <DetailDisplay label="Duration" content={weeksPipe(workPackage.duration)} paddingRight={2} />
+        <Grid item display="flex" alignItems="center" xs={12} sm={6} md={3}>
+          <ScheduleIcon sx={{ mr: 2 }} />
+          <DetailDisplay label="End Date" content={datePipe(workPackage.endDate)} paddingRight={1} />
         </Grid>
 
-        <Grid item xs={4} md={4}>
-          <DetailDisplay label="Timeline Status" content={timelinePipe(workPackage.timelineStatus)} paddingRight={2} />
+        <Grid item display="flex" alignItems="center" xs={12} sm={6} md={3}>
+          <DoneAllIcon sx={{ mr: 2 }} />
+          <DetailDisplay label="Expected Progress" content={percentPipe(workPackage.expectedProgress)} paddingRight={1} />
+        </Grid>
+
+        <Grid item display="flex" alignItems="center" xs={12} sm={6} md={3}>
+          <ScheduleIcon sx={{ mr: 2 }} />
+          <DetailDisplay label="Timeline Status" content={timelinePipe(workPackage.timelineStatus)} paddingRight={1} />
         </Grid>
       </Grid>
-    </PageBlock>
+
+      <Typography
+        variant="h5"
+        sx={{
+          mb: 1,
+          mt: 5
+        }}
+      >
+        Blocked By
+      </Typography>
+
+      <Stack direction="row" alignItems="center" divider={<Divider orientation="vertical" flexItem />} spacing={2}>
```

Comment:
why are switching to stacks here.

Reply:
all the code in details was done in another ticket, so I hadn't looked through it, I can go through and clean up any weird/ unnecessary stuff

Reply:
zach probably did it bc there's no functional difference. You can mix and match a lot of the same components to achieve the same effect, a lot of our stuff is just the same components cause it's usually one or two of us doing a lot of it at once

Reply:
also it looks like he's using this divider prop as well

Reply:
<img width="926" alt="image" src="https://github.com/Northeastern-Electric-Racing/FinishLine/assets/29521172/f91d9603-8f31-401e-8e92-7d1d0e475467">